src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else if (days < 7) {
+    return days * 40 - 20;
```

Comment:
```suggestion
    return (days * 40) - 20;
```
add parentheses or move `days * 40` to some variable